src/main/java/hlf/java/rest/client/controller/FabricOperationsController.java

```diff
@@ -25,6 +25,18 @@ public class FabricOperationsController {
 
   @Autowired private SerializationUtil serializationUtil;
 
+  /**
+   * Get the anchor peers defined in a channel.
+   *
+   * @param channelName - the name of the channel for which you will retrieve the anchor peer nodes
+   * @return ResponseEntity<ClientResponseModel> - contains the list of anchor peers in a particular
+   *     channel
+   */
+  @GetMapping(value = "/channel/{channelName}/anchor_peers")
+  public ResponseEntity<ClientResponseModel> getAnchorPeerForChannel(
```

Comment:
It is better to send `Set<String>` instead of `ClientResponseModel` in `ResponseEntity`

Reply:
Addressed.